tests/fuzzer.c

```diff
@@ -489,6 +489,53 @@ static int basicUnitTests(U32 const seed, double compressibility)
     }
     DISPLAYLEVEL(3, "OK \n");
 
+    DISPLAYLEVEL(3, "test%3d: superblock uncompressable data, too many nocompress superblocks : ", testNb++)
+    {
+        ZSTD_CCtx* cctx = ZSTD_createCCtx();
+        BYTE* src = (BYTE*)CNBuffer; BYTE* dst = (BYTE*)compressedBuffer;
```

Comment:
nit: `const`ify :
`const BYTE* const src`
`BYTE* const dst`
`size_t const srcSize`
`size_t const dstCapacity`

Reply:
dst, srcSize and dstCapacity change below but I'll const src.

Reply:
That's a good example actually !
I missed that these variables are modified inside the inner loop.

Once the ` const` property is consistently applied, the _absence_ of `const` becomes a useful signal : it means "these variables are really mutable, pay attention to them", and helps focus the reviewers/readers/future contributor's attention.